Fix typos in advanced configuration document (#569)
DataDog/dd-sdk-ios: docs/rum_collection/advanced_configuration.md

> : Enables tracking `URLSession` tasks (network requests) as RUM resources. The `firstPartyHosts` parameter defines hosts that will be categorized as `first-party` resources (if RUM feature is enabled) and will have tracing information injected (if tracing feature is enabled).

Reviewer: ```suggestion
: Enables tracking `URLSession` tasks (network requests) as RUM resources. The `firstPartyHosts` parameter defines hosts that are categorized as `first-party` resources (if RUM is enabled) and have tracing information injected (if tracing feature is enabled).
```

: Enables or disables the Tracing feature.

`setSpanEventMapper(_ mapper: @escaping (SpanEvent) -> SpanEvent)`
: Sets the data scrubbing callback for spans. This can be used to modify or drop span events before they are sent to Datadog.
 
### Automatically track views

> To automatically track views (`UIViewControllers`), use the `.trackUIKitRUMViews()` option when configuring the SDK. By default, views will be named with the view controller's class name. To customize it use `.trackUIKitRUMViews(using: predicate)` and provide your own implementation of the `predicate` which conforms to `UIKitRUMViewsPredicate` protocol:

Reviewer: ```suggestion
To automatically track views (`UIViewControllers`), use the `.trackUIKitRUMViews()` option when configuring the SDK. By default, views are named with the view controller's class name. To customize it, use `.trackUIKitRUMViews(using: predicate)` and provide your own implementation of the `predicate` which conforms to `UIKitRUMViewsPredicate` protocol:
```

    delegateQueue: nil
)
```

> This will track all requests sent with the instrumented `session`. Requests matching `example.com` domain will be marked as "first party" and tracing information will be sent to your backend to [connect the RUM resource with its Trace][10].

Reviewer: ```suggestion
This tracks all requests sent with the instrumented `session`. Requests matching the `example.com` domain are marked as "first party" and tracing information is sent to your backend to [connect the RUM resource with its Trace][10].
```

> To add custom attributes to resources, use the `.setRUMResourceAttributesProvider(_ :)` option when configuring SDK. By setting attributes provider closure you can return additional attributes to be attached to tracked resource. 

Reviewer: ```suggestion
To add custom attributes to resources, use the `.setRUMResourceAttributesProvider(_ :)` option when configuring SDK. By setting attributes provider closure, you can return additional attributes to be attached to tracked resource. 
```
